src/parachain/collateral.ts

```diff
@@ -15,17 +15,17 @@ export interface CollateralAPI {
     /**
      * @returns Total locked DOT collateral
      */
-    totalLockedDOT(): Promise<Balance>;
+    totalLockedDOT(): Promise<BN>;
```

Comment:
I'd suggest we remove the DOT part in the name since this will change on Kusama and this is anyway a breaking change for the UI.